client/galaxy/scripts/apps/admin.js

```diff
@@ -91,6 +94,17 @@ window.app = function app(options, bootstrapped) {
             );
         },
 
+        show_user_api_keys: function() {
+            // TODO refactor into a reusable Vue component display mechanism.
+            var z = document.createElement('div');
+            this.page.display(z);
+            new Vue({
```

Comment:
I think you can simplify this to 

```
new Vue(UserAPIKeys).$mount(z)
```

Which will make it easier to generalize later because you can just pass the component in, no need for template.

Reply:
Nice, trying that out now.

Reply:
@jxtx Works great, thanks!